pkg/network/common/egressip.go

```diff
@@ -238,7 +243,12 @@ func (eit *EgressIPTracker) UpdateHostSubnetEgress(hs *networkv1.HostSubnet) {
 		node.requestedCIDRs = newRequestedCIDRs
 		node.parsedCIDRs = make(map[string]*net.IPNet)
 		for _, cidr := range hs.EgressCIDRs {
-			_, parsed, _ := net.ParseCIDR(string(cidr))
+			_, parsed, err := net.ParseCIDR(string(cidr))
+			if err != nil {
+				utilruntime.HandleError(fmt.Errorf("Could not parse HostSubnet %q CIDR: %v", hs.Name, err))
```

Comment:
This shouldn't be needed since you only call this function on validated HostSubnets.

Reply:
yea true, I added it in case someone decided to call `UpdateHostSubnetEgress` from elsewhere. But, I am sure that can be handled later on, this does not serve any purpose for now